src/subscript/ri_wellmod/ri_wellmod.py

```diff
@@ -0,0 +1,532 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+"""
+ ri_wellmod.py
+
+
+"""
+import argparse
+import os.path
+import fnmatch
+import shutil
+import xml.dom.minidom
+import re
+import logging
+from datetime import datetime
+
+import rips
+
+from subscript import getLogger
+
+DESCRIPTION = """
+``ri_wellmod`` is a utility to generate Eclipse well model definitions
+(WELSPECS/WELSPECL, COMPDAT/COMPDATL, WELSEGS, COMPSEGS) using ResInsight. The script
+takes as input a ResInsight project with wells and completions defined, in addition to
+an Eclipse case (either an initialized case or an input case with grid and PERMX|Y|Z
+and NTG defined in the GRDECL format).
+
+.. note:: Well names specified as command line arguments are assumed to refer to the
+   Eclipse well names, i.e., the completion export names as defined in the ResInsight
+   wells project.
+"""
+
+CATEGORY = "modelling.reservoir"
+
+EXAMPLES = """
+.. code-blck:: console
+
+ FORWARD_MODEL RI_WELLMOD(<RI_PROJECT>=<CONFIG_PATH>/../../resinsight/input/well_modelling/wells.rsp, # noqa: E501
+                          <ECLBASE>=<ECLBASE>,
+                          <OUTPUTFILE>=<RUNPATH>/eclipse/include/schedule/well_def.sch)
```

Comment:
I guess this must be a long line if people should be able to copy and paste from documentation site into their editors.

Reply:
Added a warning that they should remove the line breaks if copying this..